### ISSUE TITLE
Change font size in the editor

### ISSUE THREAD

**Issue by Papirosnik** [defold/defold#5365]
Definitely, defold lacks such a base setting as font size.
My eyesight becomes a bit more slight, while I avoid using glasses and lenses. And working with such a small font is pretty uncomfortable.

**Comment by subsoap** [comment]
There are ways to edit the css of the editor after you download it but they are not easy to do reliably. Now that the source is available adding a size modifier within the editor's settings should be reasonable for someone to do.

**Comment by Papirosnik** [comment]
It was a big surprise for me today.
Yesterday I've updated Defold to 1.2.177 on my linux system, but got a chance to work on my just-for-fan game today only.
Just closed my previous IDE, opened Defold editor and looking at small font by force of habit have pressed Ctrl-+, Ctrl+- a couple of times.
And font size in the editor window became smaller!
Quick investigation shows that there are two menu items in the View menu: Increase/Decrease Font Sizer :+1: 
Cool!
(and BTW, shortcut (Ctrl++) for increasing font size does not work on Linux 20.10, font can be made bigger via menu item only. Ctrl+- works as expected)
And now I am not sure and wonder if those options were missing at the moment when I requested this feature )
Anyway, thank you guys.
Great job!
